ui-grid/ui-grid.d.ts

```diff
@@ -1512,6 +1512,10 @@ declare namespace uiGrid {
              */
             expandableRowHeight?: number;
             /**
+             * reference to the parent grid scope (the parent scope of the sub-grid element)
+             */
+            expandableRowScope?: Object;
```

Comment:
Good point :+1:
However, I believe `expandableRowScope` is used to pass custom properties to the expandable grid, alongwith the parent grid scope, of course. Atleast, that is the use case we have.
In that case, adding `ng.IScope` would restrict that since `tsc` wouldn't allow unknown properties (custom in this case). Please correct me if I am wrong/